cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,49 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
-  });
-
-  it('', () => {
+describe('New Article', () => {
```

Comment:
The test case should have a clear and descriptive name. Instead of 'New Article', a more descriptive name such as 'Article Creation and Deletion' would be better.